src/kirby/components/toolbar/toolbar.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../scss/web-imports';
+
+.toolbar {
+  --background:  #{get-color('background-color')};
+  --color: #{get-color('background-color-contrast')};
+}
+
+@each $color-name, $color-value in $main-colors {
+  :host(.#{$color-name}) {
+    ion-toolbar {
+      --background:  #{get-color($color-name)};
+      --color: #{get-color($color-name+'-contrast')};
+    }
+  }
+}
+
+.toolbar-item {
+
+  &-left {
```

Comment:
I know naming is hard. But we have `left` `secondary` and `end`. That gives me as reader no info what this is about.
By just renaming them to `start`, `middle` and `end` i could here see that is has something to do with positioning inside the tool bar.

Reply:
Well middle is kinda misleading since it is places to the left of the right one! I see your point and left should definitely be called start. But middle is a bad choice I think!